File: backend/services/post-rust-actix-web-fdm/src/common/api/invalid_size.rs
use crate::common::utils::error::*;

impl super::super::Size {
    pub(in crate::common) fn size_non_positive() -> ErrorBody {
        ErrorBody {
            error: ErrBody {
                error: "SIZE_NON_POSITIVE_INTEGER".into(),
                reason: "size should be a positive integer".to_string(),
                message: "size should be a positive integer".to_string(),
            },
        }
    }
}

File: backend/services/post-kt-vertx-fdm/src/main/kotlin/me/jason5lee/post_kt_vertx_fdm/common/api/InvalidSize.kt
package me.jason5lee.post_kt_vertx_fdm.common.api

import me.jason5lee.post_ktor_mongo_fdm.common.utils.Err
import me.jason5lee.post_ktor_mongo_fdm.common.utils.FailureBody

object InvalidSize {
    val nonPositiveInteger: FailureBody = FailureBody(
        error = Err(
            error = "INVALID_SIZE",
            reason = "Size must be a positive integer",
            message = "Size must be a positive integer",
        )
    )
}
